packages/tyche_utils/tyche_utils.0.1.0/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "Utilities for Tyche to visualize results produced from tests run by Mica"
+maintainer: ["Ernest Ng <ernest@cs.cornell.edu>"]
+authors: ["Ernest Ng <ernest@cs.cornell.edu>"]
+license: "ISC"
+homepage: "https://github.com/ngernest/mica"
+bug-reports: "https://github.com/ngernest/mica/issues"
+depends: [
+  "dune" {>= "3.7"}
+  "yojson"
```

Comment:
One of the errors is a "lower bounds check failure". The actual errors looks like this:

```
#=== ERROR while compiling tyche_utils.0.1.0 ==================================#
# context              2.3.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.11.2 | pinned(https://github.com/ngernest/mica/archive/refs/tags/v0.1.0.tar.gz)
# path                 ~/.opam/4.11/.opam-switch/build/tyche_utils.0.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p tyche_utils -j 39 @install
# exit-code            1
# env-file             ~/.opam/log/tyche_utils-7-a31302.env
# output-file          ~/.opam/log/tyche_utils-7-a31302.out
### output ###
# File "/home/opam/.opam/4.11/lib/ocaml-compiler-libs/bytecomp/ocaml-compiler-libs.bytecomp.dune", line 1, characters 0-0:
<snip>
# File "lib/tyche_utils/tyche_utils.ml", line 18, characters 13-27:
# 18 |   val json : Yojson.Basic.t
#                   ^^^^^^^^^^^^^^
# Error: Unbound type constructor Yojson.Basic.t
```

This is taken from the CI log https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/51b56fad1033fa4ad04b706eef5074af6f223397/variant/compilers,4.11,tyche_utils.0.1.0,lower-bounds

It indicates that the package depends on a feature of `Yojson` that is not available in some versions matching the dependency specification.

To fix this, we can add a lower bound. This is probably a safe bet, given the change log:

```suggestion
  "yojson" {>= "2.0.O"}
```

Reply:
Thanks for the info, very helpful!